### PR TITLE
added known issue

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Added information about known issues for 2.3.2

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4753) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4753) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 1]
```diff
@@ -377,13 +383,16 @@ This fix can degrade performance in deployments that implement flat catalogs. To
...
 
-`<argument name="resourceStockItem" xsi:type="object">Magento\CatalogInventory\Model\ResourceModel\Stock\Item\Proxy</argument>`. ) *Fix submitted by [Vitaliy](https://github.com/VitaliyBoyko) in pull request [21731](https://github.com/magento/magento2/pull/21731)*. [GitHub-167](https://github.com/magento/magento2/issues/167)
+  `<argument name="resourceStockItem" xsi:type="object">Magento\CatalogInventory\Model\ResourceModel\Stock\Item\Proxy</argument>`. ) *Fix submitted by [Vitaliy](https://github.com/VitaliyBoyko) in pull request [21731](https://github.com/magento/magento2/pull/21731)*. [GitHub-167](https://github.com/magento/magento2/issues/167)
 
 ### Catalog URL rewrite
 
 <!--- ENGCOM-4386-->
 * URL rewrites are no longer overwritten in multisite deployments. *Fix submitted by [Anshu Mishra](https://github.com/AnshuMishra17) in pull request [21462](https://github.com/magento/magento2/pull/21462)*. [GitHub-21329](https://github.com/magento/magento2/issues/21329)
 
+
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]:
```suggestion
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 2]
```diff
@@ -377,13 +383,16 @@ This fix can degrade performance in deployments that implement flat catalogs. To
...
+
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]:
```suggestion
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 3]
```diff
@@ -377,13 +383,16 @@ This fix can degrade performance in deployments that implement flat catalogs. To
...
+
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]:
```suggestion
```

**Pull request merged**